### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by ComLock** [amongiants/is-element#2]
It would be very nice to know which browsers this code is tested on similar to what is done here: https://github.com/maxogden/element-class

**Comment by ComLock** [comment]
BTW have you seen
http://stackoverflow.com/questions/384286/javascript-isdom-how-do-you-check-if-a-javascript-object-is-a-dom-object

Looks like there might be some ideas for improvement there.

**Comment by jongacnik** [comment]
Agreed! I haven't looked at this little module for a while (~2 years!) so I'm sure there are some nice little improvements that can be made. Thanks for putting this on the radar, I'll add it to the todos. That said, I'm also always open to pull reqs. Thx much
